src/particles/elements/All.H

```diff
@@ -32,7 +33,7 @@ namespace impactx
         None, /* must be first, so KnownElements creates a default constructor */
         ConstF, DipEdge, Drift, Multipole, NonlinearLens,
         Programmable,
-        Quad, RFCavity, Sbend, ShortRF, Sol>;
+        Quad, RFCavity, Sbend, ShortRF, Sol, PROT>;
```

Comment:
Idea for name:
```suggestion
        Quad, RFCavity, Sbend, ShortRF, Sol, PoleRot>;
```

Reply:
Prefer the name "PRot", since references may call this either a "pole-face rotation" or also a "plane rotation", and the abbreviation is used in several codes (MaryLie, ML/I, PTC).